django/api/settings.py

```diff
@@ -34,17 +34,17 @@
 
 # Application definition
 INSTALLED_APPS = [
+    'api.apps.ApiConfig',
+    'corsheaders',
+    'django_filters',
+    'django_extensions',
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.messages',
     'django.contrib.sessions',
     'django.contrib.staticfiles',
-    'api.apps.ApiConfig',
-    'django_filters',
-    'django_extensions',
     'rest_framework',
-    'corsheaders',
```

Comment:
Can we leave these installed app in the order they were before? The merge from upstream might have caused this instead of doing a rebase.

Reply:
Yeah sure.